src/registry/domain/repository.js

```diff
@@ -258,12 +262,9 @@ module.exports = function(conf) {
       )}`,
 
     getStaticFilePath: (componentName, componentVersion, filePath) =>
-      `${repository.getComponentPath(
-        componentName,
-        componentVersion
-      )}${conf.local
-        ? settings.registry.localStaticRedirectorPath
-        : ''}${filePath}`,
+      `${repository.getComponentPath(componentName, componentVersion)}${
```

Comment:
Didn't change this, but it seems prettier seems to want to change it.

Reply:
No worries, that is by design. Perhaps we should mention that in the contributing guidelines to avoid confusion :)